enterprise/backend/src/metabase_enterprise/sandbox/api/table.clj

```diff
@@ -14,12 +12,12 @@
    [metabase.util.malli.schema :as ms]
    [toucan2.core :as t2]))
 
-(mu/defn ^:private find-gtap-question :- [:maybe (ms/InstanceOf Card)]
-  "Find the associated GTAP question (if there is one) for the given `table-or-table-id` and
-  `user-or-user-id`. Returns nil if no question was found."
+(mu/defn find-sandbox-source-card :- [:maybe (ms/InstanceOf :model/Card)]
```

Comment:
I think this should be private, no?

```suggestion
(mu/defn ^:private find-sandbox-source-card :- [:maybe (ms/InstanceOf :model/Card)]
```

Reply:
Thanks, yeah, this got lost in the backport since `mu/defn-` isn't on 50